docs/preview/yamlgettingstarted-publish.md

```diff
@@ -0,0 +1,42 @@
+# YAML getting started - Publish artifact
+
+## Publish artifact step
+
+The `publishArtifact` step can be used to upload a directory, as an artifact associated with the build.
+
+## Example: Publish the artifact staging directory
+
+In the following example, the first step creates a file under the artifact staging directory.
+
+The `publishArtifact` step uploads the artifact staging directory. When the build completes, the artifact named `drop` can be downloaded from the artfacts tab on the build summary page.
+
+```yaml
+steps:
+- script: echo hello > $(build.artifactStagingDirectory)/hello.txt
+
+- publishArtifact: drop
```

Comment:
Perhaps don't require an artifact name and default to "drop" if not specified.  That's what our designer templates do with the PublishBuildArtifacts task.